app/src/main/java/camp/nextstep/edu/github/application/App.kt

```diff
@@ -0,0 +1,8 @@
+package camp.nextstep.edu.github.application
+
+import android.app.Application
+import dagger.hilt.android.HiltAndroidApp
+
+@HiltAndroidApp
+class App: Application() {
+}
```

Comment:
```suggestion
class App: Application()
```
불필요한 중괄호는 제거해보면 어떨까요?